app/models/quote.rb

```diff
@@ -8,4 +8,16 @@ class Quote < ApplicationRecord
 
   scope :visible_to, -> (current_user){ open.or(where(user: current_user))}
   scope :recent_first, -> { order("id DESC") }
+
+  def self.set_quote_of_the_day!
+    current_qotd = Quote.find_by(is_qotd: true)
+    current_qotd.update(is_qotd: false) if current_qotd
+
+    sql = "SELECT quote_id FROM bookmarks GROUP BY quote_id ORDER BY COUNT(quote_id) DESC LIMIT 1"
```

Comment:
Do you want the least bookmarked or most? and why the query string, don't you think you can build this with all the provided QueryMethods?